src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-tutorial-control-your-data.mdx

```diff
@@ -0,0 +1,567 @@
+---
+title: "Control your data with NRQL"
```

Comment:
i'm not sure this is the best title for this doc. When I read it at first, I associated 'control' with some kind of data/ingest limiting, but it looks like it's more about manipulating data. A better title might be 'Process your raw data with NRQL' or some equivalent